packages/shared/src/hooks/useTimedAnimation.ts

```diff
@@ -47,6 +47,10 @@ export const useTimedAnimation = ({
 
   const startAnimation = useCallback(
     (duration: number) => {
+      if (interval.current) {
+        window.clearInterval(interval.current);
+      }
```

Comment:
Would you maybe use `clearInterval` function we already have for this?

Reply:
Oh, right. We can use that.